politeiawww/cmsdatabase/cockroachdb/cockroachdb.go

```diff
@@ -24,6 +24,7 @@ const (
 	tableNameInvoice       = "invoices"
 	tableNameLineItem      = "line_items"
 	tableNameInvoiceChange = "invoice_change"
+	tableNameExchangeRate  = "exchange_rate"
```

Comment:
Your table names are inconsistently pluralized.  `invoices` and `line_items` are plural.  `invoice_change` and `exchange_rate` are not plural.  It doesn't really matter whether you choose plural or not plural, but they should be consistent.  The cache pluralizes table names.

Reply:
OK will pluralize all.